src/bidders/prebid/index.js

```diff
@@ -163,6 +158,31 @@ export class Prebid extends BaseBidder {
 		);
 	}
 
+	refreshBids(slotAlias) {
```

Comment:
What do you think about calling this method once (on prebid init), instead of on each `AD_SLOT_CREATED`? Based on some flag `context.get('bidders.prebid.refreshEnabled')`. It turns out that `window.pbjs.onEvent('bidWon', ...)` is going to be called once